doc/attr.xml

```diff
@@ -1439,6 +1439,43 @@ gap> ChromaticNumber(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="ChromaticNumberFunc">
+<ManSection>
+    <Func Name="ChromaticNumber" Arg="digraph, colouring_algorithm" 
+        Label="for a digraph and colouring algorithm"/>
+    <Returns>A non-negative integer.</Returns>
+    <Description>
+      Alternative implementation of chromatic number
+      (See <Ref Attr="ChromaticNumber"/>) that allows the specific algorithm 
+      used to be selected. 
+
+      If <A>digraph</A> is a digraph without loops (see <Ref
+      Prop="DigraphHasLoops"/>), then <C>ChromaticNumber</C> uses
+      <A>colouring algorithm</A> (permitted values for this are given below)
+      to find the least non-negative integer <C>n</C> such that there is a
+      proper colouring of <A>digraph</A> with <C>n</C> colours. See
+      <Ref Attr="ChromaticNumber"/> for the definition of a proper colouring.
+
+      The permitted implementations, and values for <A>colouring_algorithm</A>,
+      are:
+      <List>
+          <Item><C>DigraphColouringAlgorithmLawler</C> - Lawler's Algorithm 
+          <Cite Key="Law1976"/></Item>
+          <Item><C>DigraphColouringAlgorithmByskov</C> - Byskov's Algorithm 
+          <Cite Key="Bys2002"/></Item>
+      </List>
```

Comment:
This documentation could then be added to the main bit of documentation, perhaps like:
> it is possible to select the algorithm used to compute the chromatic number of a digraph via ...
> The permitted algorithms, and values for BLAH, are...